pyam/iiasa.py

```diff
@@ -648,16 +660,24 @@ def lazy_read_iiasa(
         If `True`, include all meta categories & quantitative indicators
         (or subset if list is given).
     creds : str or :class:`pathlib.Path`, optional
-        | Credentials (username & password) are not required to access
-          any public Scenario Explorer instances (i.e., with Guest login).
-        | See :class:`pyam.iiasa.Connection` for details.
-        | Use :meth:`pyam.iiasa.set_config` to set credentials
-          for accessing private/restricted Scenario Explorer instances.
+        Path to a file with authentication credentials. This feature is deprecated,
+        please run ``ixmp4 login <username>`` in a console instead.
     base_url : str
         Authentication server URL
     kwargs
-        Arguments for :meth:`pyam.iiasa.Connection.query`
+        Arguments for :meth:`pyam.read_ixmp4` or :meth:`pyam.iiasa.Connection.query`.
+
+    Notes
+    -----
+    This feature does currently not support reading data from |ixmp4| platforms.
+
+    Credentials (username & password) are not required to access any public |ixmp4|
+    or Scenario Explorer database (i.e., with Guest login).
     """
+    if name in [i.name for i in ixmp4.conf.settings.manager.list_platforms()]:
```

Comment:
For better readability I always prefer to use the first letter of the object or better the full word:
```suggestion
    if name in [platform.name for platform in ixmp4.conf.settings.manager.list_platforms()]:
```